lib/index.js

```diff
@@ -69,7 +71,7 @@ Adapter.prototype.set = function set(id, room, fn) {
 Adapter.prototype.get = function get(id, fn) {
   var rooms = id ? this.sids[id] || {} : this.rooms;
   rooms = Object.keys(rooms);
-  if (fn) process.nextTick(function tick() {
+  if (fn) setImmediate(function immediate() {
```

Comment:
You are right, I'll update.
Do you want to use `null` for the error? It will be even cleaner if we use `undefined`:
```js
if (fn) setImmediate(fn);
```